packages/pharos/package.json

```diff
@@ -84,7 +84,22 @@
     "sinon": "^12.0.1",
     "style-dictionary": "^3.0.1",
     "ts-lit-plugin": "^1.2.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.9.4"
   },
-  "customElements": "custom-elements.json"
+  "customElements": "custom-elements.json",
+  "exports": {
```

Comment:
Think you'll want to add a changeset and designate it as [a breaking change](https://nodejs.org/api/packages.html#package-entry-points) since individual component imports will be invalid.